cli/main.c

```diff
@@ -780,6 +780,108 @@ static int temp(int argc, char **argv)
 	return 0;
 }
 
+static int port_bind_info(int argc, char **argv)
+{
+	const char *desc = "Bind info for physical port";
+	float ret;
+	struct switchtec_bind_status_out bind_status;
+	static struct {
+		struct switchtec_dev *dev;
+		int phy_port;
+	} cfg = {};
+	const struct argconfig_options opts[] = {
+		DEVICE_OPTION,
+		{"physical", 'f', "", CFG_INT, &cfg.phy_port, required_argument,
+			"physical port number"},
+		{NULL}};
+
+
+
```

Comment:
Please don't use multiple blank lines. One is sufficient. (This follows Linux Kernel coding convention). There are a couple cases below of this too please fix them all.